easybuild/easyconfigs/l/libdrm/libdrm-2.4.125-GCCcore-14.3.0.eb

```
@@ -0,0 +1,34 @@
+easyblock = 'MesonNinja'
+
+name = 'libdrm'
+version = '2.4.125'
+
+homepage = 'https://dri.freedesktop.org'
+description = "Direct Rendering Manager runtime library."
```

```suggestion
description = """Direct Rendering Manager runtime library."""
```

Messed up by `--try-update-deps`